elasticsearch.md

```diff
@@ -799,11 +799,86 @@ So that Kibana will forward the necessary headers to Elasticsearch.
 
 [Impersonation](details/impersonation.md) is supported by this rule without an extra configuration.
 
-#### `groups_or` (or `groups`)
+#### `user_belongs_to_groups`
 
-`groups_or: ["group1", "group2"]`
+The ACL block will match when the user belongs to groups matching the condition defined in the `user_belongs_to_groups` config section. The information about what users belong to what groups is defined in the `users` section, typically situated after the ACL, further down in the YAML.
 
-The ACL block will match when the user belongs to any of the specified groups. The information about what users belong to what groups is defined in the `users` section, typically situated after the ACL, further down in the YAML.
+##### `any_of`
+
+The ACL block will match when the user belongs to any of the specified groups.
+
+```yaml
+  user_belongs_to_groups:
+    any_of: ["group1", "group2"]
+```
+
+With deprecated syntax (without being wrapped inside the `user_belongs_to_groups` section):
```

Comment:
In general, we don't mention the deprecated syntax. We do support it, but we want our users to migrate to the new one.